lib/libutee/include/tee_api_types.h

```diff
@@ -162,6 +162,21 @@ typedef struct {
 	uint32_t millis;
 } TEE_Time;
 
+/* I2C Api API */
```

Comment:
Why in this file?

Reply:
only because I noticed TEE_Time there (I guess makes no sense). shall I create its own tee_i2c.h as done with tee_tcpsocket?

Reply:
I assume that nothing of this is exposed to secure user space, so I'd suggest `core/arch/arm/include/kernel/rpc_i2c.h`.

Reply:
By the way, `tee_api_types.h` only provides whats specified in GlobalPlatform Core Internal API.